config/listflag_test.go

```diff
@@ -107,4 +107,16 @@ func TestListFlag(t *testing.T) {
 			t.Error("unexpected string representation from yaml", output, input)
 		}
 	})
+
+	t.Run("unmarshal error", func(t *testing.T) {
+		const input = `---
+foo: foo=bar
+bar: baz
+`
+		current := commaListFlag()
+		if err := yaml.Unmarshal([]byte(input), current); err == nil {
+			t.Errorf("Failed to get error from Unmarshal() for invalid input: %q", input)
+		}
+
+	})
```

Comment:
Maybe use `this is not yaml` as an input to make it obvious what is the tested problem

Reply:
fixed